modules/network-observability-includelist-example.adoc

```diff
@@ -19,7 +19,7 @@ You can create custom Prometheus rules for the Netobserv dashboard metrics to tr
 [source,yaml]
 ----
 apiVersion: monitoring.coreos.com/v1
-kind: PrometheusRule
+kind: AlertingRule
 metadata:
   name: netobserv-alerts
   namespace: openshift-netobserv-operator
```

Comment:
Ok it turns out the namespace has to be `openshift-monitoring`!
```suggestion
  namespace: openshift-monitoring
```

Reply:
Hmm okay. Is this okay to change to openshift-monitoring? Like we don't specifically want/need it to be openshift-netobserv-operator, do we?